typing/typemod.ml

```diff
@@ -355,11 +359,14 @@ let rec approx_modtype env smty =
       raise (Error_forward (Builtin_attributes.error_of_extension ext))
 
 and approx_module_declaration env pmd =
-  {
-    Types.md_type = approx_modtype env pmd.pmd_type;
-    md_attributes = pmd.pmd_attributes;
-    md_loc = pmd.pmd_loc;
-  }
+  Builtin_attributes.warning_scope pmd.pmd_attributes
```

Comment:
I think it would be better to remove the warning scope here: the only entry point of the `approx_*` function is the `approx_modtype` function, and we agreed before that there should be no warnings here.

Reply:
Thanks, done.